examples/textual_inversion/textual_inversion.py

```diff
@@ -84,32 +85,35 @@
 logger = get_logger(__name__)
 
 
-def save_model_card(repo_id: str, images=None, base_model=str, repo_folder=None):
+def save_model_card(repo_id: str, image_logs: dict =None, base_model: str=None, repo_folder: str=None):
     img_str = ""
-    for i, image in enumerate(images):
-        image.save(os.path.join(repo_folder, f"image_{i}.png"))
+    for i, log in enumerate(image_logs):
+        images = log["images"]
+        validation_prompt = log["validation_prompt"]
+        validation_image = log["validation_image"]
+        validation_image.save(os.path.join(repo_folder, "image_control.png"))
```

Comment:
Why are you changing these lines?

Reply:
Hi @sayakpaul, I just want to know do we only need images and not validation_prompts in case of textual_inversions? I have used the validation_prompts as well that's why added this code.

Reply:
That update shouldn't be necessary.

Reply:
If we only want to use images then I think what needs to be changed and I will do that.

Reply:
Hi @sayakpaul, soo the `image.save()` function should remain as it is in the code right? And the validation_prompts are just for generating the outputs in collaboratory right?